public/app/features/dashboard/api/ResponseTransformers.test.ts

```diff
@@ -1,19 +1,52 @@
 import { DashboardV2Spec } from '@grafana/schema/dist/esm/schema/dashboard/v2alpha0/dashboard.gen';
-import { DashboardDTO } from 'app/types';
+import {
+  AnnoKeyCreatedBy,
+  AnnoKeyDashboardId,
+  AnnoKeyFolder,
+  AnnoKeySlug,
+  AnnoKeyUpdatedBy,
+  AnnoKeyUpdatedTimestamp,
+} from 'app/features/apiserver/types';
+import { DashboardDataDTO, DashboardDTO } from 'app/types';
 
 import { ResponseTransformers } from './ResponseTransformers';
 import { DashboardWithAccessInfo } from './types';
 
 describe('ResponseTransformers', () => {
   describe('v1 transformation', () => {
```

Comment:
```suggestion
  describe('v1 -> v2 transformation', () => {
```
Small nit, just to avoid confusion using symbol navigation 😄 

Same here
https://github.com/grafana/grafana/pull/97953/files#diff-b7097f373b2660e4b980ad148346780340b0cf82f6f654b7ea194884e359e186R115

Reply:
added d222c8122fadd2b41dc05617197bf158f38df646